surveyor.py

```diff
@@ -37,8 +43,11 @@
 @click.version_option(current_version)
 @click.pass_context
 def cli(ctx, prefix, hostname, profile, days, minutes, product, username, iocfile, ioctype, query, output, defdir,
-        deffile):
+        deffile, creds):
 
+    if product == "defender" and creds is None: 
+        #TODO: Make this error message RED 
```

Comment:
⛏️ Is this a stale todo here?  Looks like the raised exception message has some color associated

Reply:
Yep! Thanks for pointing that out.